public/js/use.js

```diff
@@ -288,7 +288,7 @@ function setupUse(params) {
           var image = new Image();
           image.src = reader.result;
           image.onload = function () {
-            var resizedImage = (data.files[0]['size'] > 2000000) ? resize(image, 800) : this.src;
+            var resizedImage = resize(image, 800);
```

Comment:
I'm pretty sure that we should not resize images that are < 2MB. This will fix the image orientation but it will break the scenario where there image is > 800x800 but < 2MB.